grblas/__init__.py

```diff
@@ -33,3 +63,11 @@ def init(backend='suitesparse', blocking=True):
     BinaryOp._initialize()
     Monoid._initialize()
     Semiring._initialize()
+
+    globals()["Matrix"] = Matrix
+    globals()["Vector"] = Vector
+    globals()["Scalar"] = Scalar
+    globals()["UnaryOp"] = UnaryOp
+    globals()["BinaryOp"] = BinaryOp
+    globals()["Monoid"] = Monoid
+    globals()["Semiring"] = Semiring
```

Comment:
Why the need for this?  Declaring these variable names as `global` above should make this unnecessary.

Reply:
Fixed